go.mod

```diff
@@ -32,7 +32,7 @@ require (
 )
 
 require (
-	github.com/aquasecurity/trivy v0.35.0
+	github.com/aquasecurity/trivy v0.31.3
```

Comment:
Why did you need to degrade Trivy?

Reply:
wasn't intentionally, happened after i ran go get for the kms package. after rebasing to latest `main` it's not an issue anymore 😄